public/style.css

```diff
@@ -1,3 +1,16 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Abril+Fatface&display=swap');
+
+*{
+  color: #112D4E;
```

Comment:
This looks like a really high level style assignment. This would potentially affect a lot of elements on the page because of `*` selector.

We can give this colour property to the specific element which we would like it to be applied on instead of using here.